src/WindowsAzureMessaging/WindowsAzureMessaging/Helpers/MSInstallationManager.m

```diff
@@ -13,7 +13,6 @@
 #import <UIKit/UIKit.h>
 
 // TODO: Move to loading from constants file
```

Comment:
You can probably remove this TODO

Reply:
There are still constants left here to be moved to constants file. It is done in the PR https://github.com/Azure/azure-notificationhubs-ios/pull/70